src/storage/pooled_storage_manager.h

```diff
@@ -54,10 +54,11 @@ class GPUPooledStorageManager final : public StorageManager {
   /*!
    * \brief Default constructor.
    */
-  GPUPooledStorageManager() {
+  GPUPooledStorageManager(int dev_id) {
```

Comment:
Maybe pass the context instead? And call it creation_ or initial_context? And maybe add a short explanation to the parameters documentation?

Reply:
Done.